### PR TITLE
Issue 691 - Update plotlyjs

### LINKED ISSUE #691: Upgrade plotly.js to 1.51.1
https://github.com/plotly/plotly.js/releases/tag/v1.51.0

Particularly, I'd like to use `title.standoff` in Dash Design Kit.

### PR DESCRIPTION
**Pull request by Marc-Andre-Rivet**
Fixes #691

### CODE REVIEW

File: tests/integration/calendar/test_date_picker_single.py [hunk 1]
```diff
@@ -114,9 +114,7 @@ def cb(clicks):
     switch.click()
 
     memorized = dash_dcc.select_date_single("dps-memory", day="4")
-    amnesiaed = dash_dcc.select_date_single(
-        "dps-none", day="1", outside_month=True
-    )
+    amnesiaed = dash_dcc.select_date_single("dps-none", day="11")
```
Comment (Marc-Andre-Rivet) [inline tests/integration/calendar/test_date_picker_single.py]: Interesting month rollover bug in this test. November 2019 ends on a Saturday, there is no 1st day of next month in this calendar! This is the only instance where we are checking for "out-of-month" days in dcc tests. Out-of-month is not important here as we are really just testing persistence.

image

Comment (Marc-Andre-Rivet) [reply tests/integration/calendar/test_date_picker_single.py]: Funny enough, https://github.com/plotly/dash-core-components/pull/692/commits was not affected even with the last commit happening in November b/c the test does `-timedelta(days=1)`

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+### Updated
+- [#693](https://github.com/plotly/dash-core-components/pull/693) Upgraded plotly.js to 1.51.1
+  - [Feature release 1.51.0](https://github.com/plotly/plotly.js/releases/tag/v1.51.0) which contains:
```
Comment (Marc-Andre-Rivet) [inline CHANGELOG.md]: Needs the significant tidbits

Comment (alexcjohnson) [reply CHANGELOG.md]:
```suggestion
  - [Feature release 1.51.0](https://github.com/plotly/plotly.js/releases/tag/v1.51.0) which contains:
    - A new `image` trace type to display 3- or 4-channel color images as data
    - `automargin` for `pie` charts for better readability when labeling lots of small slices
    - Toggle-type `updatemenus`
    - `zh-CN` localization
    - And various other small features and bug fixes
```

**Pull request merged**